### PR TITLE
[VEN-2723]: Deployments of Oracles on OpSepolia

### PR DESCRIPTION
**Pull request by Debugger022**
## Description
Resolves #

This PR contains the deployments of **Oracles** on **OpSepolia** along with the config for `OpSepolia` and `OpMainnet`.
```
Deployed Addresses:
    "BoundValidator": "0x482469F1DA6Ec736cacF6361Ec41621f811A6800",
    "ChainlinkOracle_Proxy": "0x493C3f543AEa37EefF17D823f27Cb1feAB9f3143",
    "DefaultProxyAdmin": "0xa9aaf2A1cCf2C3a87997942abaA740887cC89241",
    "ResilientOracle": "0x6c01ECa2B5C97F135406a3A5531445A7d977D28e",
```

**Note:** Need to update package version for governance-contracts once new package is released.

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: helpers/deploymentConfig.ts [hunk 1]
```diff
@@ -1015,6 +1025,9 @@ export const assets: Assets = {
       oracle: "redstone",
     },
   ],
+  opsepolia: [
+    // TODO: add tokens
+  ],
```
Comment (chechu) [inline helpers/deploymentConfig.ts:1030]: We need to deploy first the mock tokens in the IL repo

File: .env.example [hunk 2]
```diff
@@ -17,6 +17,8 @@ ARCHIVE_NODE_ethereum=https://eth-mainnet.nodereal.io/v1/<YOUR_KEY_HERE>
 #ARCHIVE_NODE_arbitrumone=https://open-platform.nodereal.io/<YOUR_KEY_HERE>/arbitrum-nitro/
 #ARCHIVE_NODE_zksyncsepolia=https://zksync-sepolia.g.alchemy.com/v2/<YOUR_KEY_HERE>
 #ARCHIVE_NODE_zksyncmainnet=https://open-platform.nodereal.io/<YOUR_KEY_HERE>/zksync
+#ARCHIVE_NODE_opsepolia="https://sepolia.optimism.io"
+#ARCHIVE_NODE_opmainnet="https://opt-mainnet.nodereal.io/v1/<YOUR_KEY_HERE>"
```
[the remark this answers is not in the document]
Comment (Debugger022) [reply .env.example]: Removed, 4d08d3c28e769ecbc77a6509c19cc0764e432505

File: helpers/deploymentConfig.ts (hunk 1 above)
Comment (Debugger022) [reply helpers/deploymentConfig.ts:1034]: Will update the addresses here once tokens are deployed.

**Comment by GitGuru7** [comment]
Add token [details](https://github.com/VenusProtocol/oracle/pull/204#discussion_r1741867239) in config, otherwise LGTM!

File: helpers/deploymentConfig.ts [hunk 3]
```diff
@@ -290,6 +297,14 @@ export const chainlinkFeed: Config = {
...
     USDT: "0xB615075979AE1836B476F651f1eB79f0Cd3956a9",
   },
+  opsepolia: {
+    WBTC: "0x9f5039a86AF12AB10Ff16659eA0885bb4C04d013",
+    WETH: "0x4200000000000000000000000000000000000006",
+    USDC: "0x71B49d40B10Aa76cc44954e821eB6eA038Cf196F",
+    USDT: "0x9AD0542c71c09B764cf58d38918892F3Ae7ecc63",
+    OP: "0xEC5f6eB84677F562FC568B89121C5E5C19639776",
+    XVS: "0xF020C751efb29922075CE7A22d75bF310b20D71c",
+  },
```
Comment (chechu) [inline helpers/deploymentConfig.ts:307]: These are the chainlink feeds, not the token addresses.

```suggestion
  opsepolia: {
    WBTC: "0x3015aa11f5c2D4Bd0f891E708C8927961b38cE7D",
    WETH: "0x61Ec26aA57019C486B10502285c5A3D4A4750AD7",
    USDC: "0x6e44e50E3cc14DD16e01C590DC1d7020cb36eD4C",
    USDT: "0xF83696ca1b8a266163bE252bE2B94702D4929392",
    OP: "0x8907a105E562C9F3d7F2ed46539Ae36D87a15590",
  },
```

File: helpers/deploymentConfig.ts [hunk 4]
```diff
@@ -1015,6 +1030,39 @@ export const assets: Assets = {
...
+      oracle: "chainlink",
+    },
+    {
+      token: "WETH",
+      address: "0x4200000000000000000000000000000000000006",
+      oracle: "chainlink",
+    },
+    {
+      token: "XVS",
+      address: "0xF020C751efb29922075CE7A22d75bF310b20D71c",
```
Comment (chechu) [inline helpers/deploymentConfig.ts:1061]:
```suggestion
      address: "0x789482e37218f9b26d8D9115E356462fA9A37116",
```

File: helpers/deploymentConfig.ts (hunk 3 above)
Comment (Debugger022) [reply helpers/deploymentConfig.ts]: My bad 😓 , I have reverted the changes: 67b3129ad5922de430d8bb5dcb539d7e6538afb8

File: helpers/deploymentConfig.ts (hunk 4 above)
Comment (Debugger022) [reply helpers/deploymentConfig.ts]: Updated, 67b3129ad5922de430d8bb5dcb539d7e6538afb8

**Review by chechu — APPROVED** [review]

**Pull request merged**